src/com/lftechnology/java/training/sanish/application/component/UserLogin.java

```diff
@@ -0,0 +1,92 @@
+package com.lftechnology.java.training.sanish.application.component;
+
+import com.lftechnology.java.training.sanish.application.model.dao.UserDao;
+import com.lftechnology.java.training.sanish.application.model.domain.User;
+import com.lftechnology.java.training.sanish.application.utility.UserInput;
+
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * User login component
+ *
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public class UserLogin {
+    private static final Logger LOGGER = Logger.getLogger(UserLogin.class.getName());
+    private static String userName;
+    private static String password;
+    private static boolean isLogin = false;
+    private static User currentUser = null;
+
+    private UserLogin() {
+    }
+
+    public static User getCurrentUser() {
+        return currentUser;
+    }
+
+    public static boolean isLogin() {
+        return isLogin;
+    }
+
+    public static void setIsLogin(boolean isLogin) {
+        UserLogin.isLogin = isLogin;
+    }
+
+    public static void setCurrentUser(User currentUser) {
+        UserLogin.currentUser = currentUser;
+    }
+
+    /**
+     * Request username & password for login and return true if login success else false.
+     *
+     * @param inputScanner {@link Scanner}
+     * @return {@link Boolean}
+     * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+     */
+    public static boolean loginUser(Scanner inputScanner) {
+        getCredential(inputScanner);
+        UserDao userDao = new UserDao();
+        User user = userDao.getUserByUserName(userName);
+        if (user != null) {
+            if (validateUser(user)) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1066)